library/src/main/kotlin/org/codice/compliance/report/Report.kt

```diff
@@ -134,14 +146,30 @@ object Report {
         val file = File(REPORT_FILE)
         file.printWriter().use { writer ->
             Section.values().forEach { section ->
-                writer.print("\t".repeat(section.level))
-                writer.print(section.title)
-
-                // the top level section will not have a status except GENERAL and SCHEMA
-                if (section.level == 2 || section == GENERAL || section == SCHEMA) {
-                    printExceptions(section, writer)
-                } else {
-                    writer.println()
+                when (section) {
+                    CORE_3_3 -> {
```

Comment:
✏️ I don't like these hard-coded assumptions. Seems better if `section` had an inclusion or exclusion list of profiles so that in the future we could say "section x.y.z only applies to Keycloak", etc.

Reply:
Is this a good solution? https://github.com/codice/saml-conformance/pull/211/commits/d4c93dacc0bc96eee381440395769ec86a90afe1


This is what will happen now:

**When the DDF profile is NOT run:**
![screen shot 2019-01-08 at 8 28 39 am](https://user-images.githubusercontent.com/29550062/50842642-5e19a480-1324-11e9-8fd1-570ddb5e45e8.png)

**When the DDF pprofile is run:**
![screen shot 2019-01-08 at 8 29 16 am](https://user-images.githubusercontent.com/29550062/50842643-5eb23b00-1324-11e9-86b0-ed8067f220ac.png)

Reply:
👌 Ahh that's a good idea

Reply:
Alright so now it will be printed regardless but gray if we're not running the DDF profile: https://github.com/codice/saml-conformance/pull/211/commits/842011d7285cfe58fd58dfd54639cade5919105d

![screen shot 2019-01-08 at 9 45 58 am](https://user-images.githubusercontent.com/29550062/50845591-7ab8db00-132a-11e9-8807-4431c8f23360.png)

Reply:
💯

Reply:
💯